addons/website/static/src/components/configurator/configurator.js

```diff
@@ -463,12 +473,23 @@ async function getInitialState() {
         features[feature.id].website_config_preselection = wtp ? wtp.split(',') : [];
     });
 
+    // Palette color used by default as background color for menu and footer.
+    // Needed to build the recommended palette.
+    const defaultColors = {};
+    CUSTOM_BG_COLOR_ATTRS.forEach((attr) => {
+        const color = weUtils.getCSSVariableValue(`default-${attr}-bg`, style);
```

Comment:
Still not using the right convention: https://github.com/odoo/odoo/pull/75836#discussion_r702786303

Reply:
oups :eyes:  . I just changed